content/en/monitors/create/configuration.md

```diff
@@ -142,6 +174,18 @@ For some metrics that report periodically, it may make sense for triggered alert
 
 In most cases this setting is not useful because you only want an alert to resolve once it is actually fixed. So, in general, it makes sense to leave this as `[Never]` so alerts only resolve when the metric is above or below the set threshold.
 
+#### Group retention time
+
+Drop the group from the monitor status after `N` hours (can be set up to 72 hours).
```

Comment:
We need to express here that the group is dropped only if it doesn't report data